spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkV2Filters.java

```diff
@@ -245,6 +274,70 @@ public static Expression convert(Predicate predicate) {
     return null;
   }
 
+  private static <T> UnboundPredicate<T> handleComparePredicate(
```

Comment:
This is hard to review because there are a lot of aesthetic changes. Can you roll back any changes that aren't strictly necessary?

Reply:
Updated